### PR TITLE
fixPhonebookEntries

### PR DESCRIPTION
**Pull request by mahibi**
Signed-off-by: Marcel Hibbe <dev@mhibbe.de>

### CODE REVIEW

File: app/src/main/java/com/nextcloud/talk/jobs/ContactAddressBookWorker.kt [hunk 1]
```diff
@@ -212,147 +200,176 @@ class ContactAddressBookWorker(val context: Context, workerParameters: WorkerPar
...
+                            .withValueBackReference(ContactsContract.Data.RAW_CONTACT_ID, 0)
+                            .withValue(ContactsContract.Data.MIMETYPE, ContactsContract.CommonDataKinds.StructuredName.CONTENT_ITEM_TYPE)
+                            .withValue(ContactsContract.CommonDataKinds.StructuredName.DISPLAY_NAME, displayName)
+                            .build())
+                    ops.add(ContentProviderOperation
+                            .newInsert(dataUri)
+                            .withValueBackReference(ContactsContract.Data.RAW_CONTACT_ID, 0)
+                            .withValue(ContactsContract.Data.MIMETYPE, "vnd.android.cursor.item/vnd.com.nextcloud.talk2.chat")
+                            .withValue(ContactsContract.Data.DATA1, cloudId)
+                            .withValue(ContactsContract.Data.DATA2, "Chat via " + accountName)
```
Comment (nickvergessen) [inline app/src/main/java/com/nextcloud/talk/jobs/ContactAddressBookWorker.kt:313]: Not translated?

Comment (mahibi) [reply app/src/main/java/com/nextcloud/talk/jobs/ContactAddressBookWorker.kt]: i tried but had some strange problems when i modified strings.xml :D but now it worked..

**Pull request merged**